x-pack/plugin/profiling/src/main/java/org/elasticsearch/xpack/profiling/GetStackTracesRequest.java

```diff
@@ -42,6 +42,7 @@ public class GetStackTracesRequest extends ActionRequest implements IndicesReque
     public static final ParseField CUSTOM_CO2_PER_KWH = new ParseField("co2_per_kwh");
     public static final ParseField CUSTOM_DATACENTER_PUE = new ParseField("datacenter_pue");
     public static final ParseField CUSTOM_PER_CORE_WATT = new ParseField("per_core_watt");
```

Comment:
I think we should rename the public facing field for consistency to include something like `x86` or `x86_64`. Even if means more work now, the API changes are not yet released and we should strive to avoid surprises in the future.

Reply:
Currently, this value is a default to anything not being ARM64. Even if the description/naming in Kibana suggest it is an x86 specific value.
But I get your point, will get in touch with @cauemarcondes .

Reply:
We agreed on renaming to `per_core_watt_x86`.